contributing/documentation.md

```diff
@@ -0,0 +1,73 @@
+(contributing-to-docs)=
+# Contributing to Charmed HPC's documentation
+
+The documentation for Charmed HPC is open source and welcomes community contributions. Please read through the following guidelines to best prepare yourself for contributing. If you have questions, feel free to ask them in the [Ubuntu High-Performance Computing Matrix chat](https://matrix.to/#/#hpc:ubuntu.com) or in [Charmed HPC's GitHub Discussions](https://github.com/orgs/charmed-hpc/discussions). 
+
+## Prerequisites
+
+There are a couple prerequisites to contributing to Charmed HPC's documentation:
+
+* **GitHub Account:** The Charmed HPC project uses GitHub to host its documentation. You will need an active GitHub account to report issues and provide contributions. See [Creating an account on GitHub](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github) for more details.
+
+* **Code of Conduct:** You must read and follow the Ubuntu [Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct) to contributed to Charmed HPC. Maintainers reserve the right to delete or remove any contributions that do not respect the Code of Conduct. 
+
+## Report an issue
+
+To report an error in spelling, grammar, content, or documentation code functionality, [file an issue](https://github.com/charmed-hpc/docs/issues) in Charmed HPC's bug tracker on GitHub.
+
+## Simple update
+
+The easiest way to make a quick update, especially for those new to git and GitHub, is to use GitHub's [file editor](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository) through your web browser. 
+
+## Large contribution
+
+For larger, more involved contributions, and those familiar with git and the command line, follow the [fork-and-branch](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/) process. Be sure to familiarize yourself with the [Documentation structure](doc-structure) prior to any significant work. 
+
+## Test your contribution
+
+To ensure that your contributions meet expectations and pass CI rules, check that they pass the repository's tests.
+
+### Test locally
+
+Install `npm` using the appropriate method for your operating system through one of the follow processes:
+* Through your preferred package manager
+* By following the [node version manager installation process](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-version-manager-to-install-nodejs-and-npm)
+* For Debian and Ubuntu Linux distributions: `sudo apt install npm`
```

Comment:
`snap install` instead of `apt install`?? :pleading_face::point_right::point_left: 

```suggestion
* For Ubuntu and other snap-supported Linux distributions: `sudo snap install node --classic`
```